contracts/BondFactory.sol

```diff
@@ -120,8 +132,27 @@ contract BondFactory is AccessControl {
         address collateralToken,
         uint256 collateralRatio,
         uint256 convertibleRatio,
-        uint256 maxSupply
+        uint256 maxBonds
     ) external onlyIssuer returns (address clone) {
+        if (maxBonds == 0) {
+            revert MaxBondsCantBeZero();
```

Comment:
having a contraction in the error seems weird to me. 
`MaxBondsCannotBeZero` `ZeroBondsToMint` `MustMintBonds`